.github/workflows/validate-core-vpc-configuration.yml

```diff
@@ -0,0 +1,21 @@
+name: "Validate core-vpc configuration"
+
+on: 
```

Comment:
Is it worth adding a `workflow_dispatch` trigger ([example](https://github.com/ministryofjustice/modernisation-platform/blob/main/.github/workflows/check-links.yml#L4)) so we can run this manually via GitHub actions?

Reply:
added